### PR TITLE
feat(icon): export icons for compose

### PR DESCRIPTION
**Pull request by joheredi**
This PR exports the following icons in the Teams theme:
image

- bold
- code-snippet
- emoji
- italic
- link
- meeting-new
- number-list
- quote
- sticker
- table
- underline

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Export `call-recording` SVG icon @Bugaa92 ([#585](https://github.com/stardust-ui/react/pull/585))
 - Export `canvas-add-page` SVG icon @priyankar205 ([#601](https://github.com/stardust-ui/react/pull/601))
 - Add `sizeModifier` variable (with `x` and `xx` values) to `Icon`'s Teams theme styles @priyankar205 ([#601](https://github.com/stardust-ui/react/pull/601))
+- Export icons for compose @joheredi ([#607](https://github.com/stardust-ui/react/pull/607))
```
Comment (mnajdova) [inline CHANGELOG.md]: Please mention the Teams theme for changes related only for this theme

**Comment by mnajdova** [comment]
Also, please take a look into the screener changed, and ensure that only the new icons are added and there is no regression to the previos icons.

**Comment by kuzhelov** [comment]
please, note that you have incorrect target to merge into. Ensure that you have `master` (not `stardust:master`) branch for your changes (probably, you will need to create another PR and reapply your changes there):

image

File: src/themes/teams/components/Icon/svg/icons/bold.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import { TeamsProcessedSvgIconSpec } from '../types'
+
+export default {
```
Comment (johannao76) [inline src/themes/teams/components/Icon/svg/icons/bold.tsx:4]: Isn't this basically a duplicate of the one in ProcessedIcons folder? Why do we need the dupes?

**Review by johannao76 — APPROVED** [review]
:shipit:

File: src/themes/teams/components/Icon/svg/icons/bold.tsx (hunk 2 above)
Comment (joheredi) [reply src/themes/teams/components/Icon/svg/icons/bold.tsx:4]: I'm following the template for exporting Icons: https://github.com/stardust-ui/react/pull/585

@Bugaa92, what's the reason behind duplicating the files?

**Comment by joheredi** [comment]
> please, note that you have incorrect target to merge into. Ensure that you have `master` (not `stardust:master`) branch for your changes (probably, you will need to create another PR and reapply your changes there):
> 
> image

@kuzhelov I think it shows as stardust-ui:master because I'm merging from my Fork. I'm not an "official" contributor so I couldn't create a branch within the main fork.

**Comment by kuzhelov** [comment]
> @kuzhelov I think it shows as stardust-ui:master because I'm merging from my Fork. I'm not an "official" contributor so I couldn't create a branch within the main fork.

@joheredi, now this omission is fixed :)

**Comment by joheredi** [comment]
Submitting a new PR against master

**Pull request closed without merging**